clang/lib/DPCT/CallExprRewriterCommon.h

```diff
@@ -0,0 +1,1741 @@
+#ifndef DPCT_CALL_EXPR_REWRITER_COMMON_H
+#define DPCT_CALL_EXPR_REWRITER_COMMON_H
+
```

Comment:
Maybe , we can create CallExprRewriterCommon.cpp , and put the implementation in the source file. When the specific implementation is modified, all source include this header file would need not be recompiled.